.gitignore

```diff
@@ -1,4 +1 @@
-swarmvars.json
-set_secrets.sh
-*.pyc
```

Comment:
These files should not up on you machine and i like `*ignore` files to be short. But not too strong opinion on this in case you insist.